src/WarpedPsf.cc

```diff
@@ -148,6 +134,47 @@ std::shared_ptr<afw::detection::Psf::Image> warpAffine(afw::detection::Psf::Imag
     return ret;
 }
 
+// A helper struct for logic and intermediate results used by both
+// doComputeKernelImage and doComputeBBox.
+struct PreparedTransforms {
```

Comment:
Very nitpicky comment, but PreparedTransforms doesn't quite tell me what it is doing. I don't have an alternative suggestion, and not user-facing, it should be totally okay to keep the name, but a comment on what is being prepared would help.